.github/workflows/docker-build.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   assets:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Why not `ubuntu-22.04` for the sake of consistency? Isn't `ubuntu-24.04` the latest LTS release?